packages/core/src/login/ILoginOptions.ts

```diff
@@ -47,4 +47,9 @@ export default interface ILoginOptions extends ILoginInputOptions {
   prompt?: string;
   // Force the token type to be required (i.e. no longer optional).
   tokenType: "DPoP" | "Bearer";
+  /**
+   * This callback will be called if, during the refresh token flow, the refresh token is rotated by the Solid Identity
+   * Provider. In this case, the provided function is called with the new token as a parameter.
+   */
+  handleRefreshTokenRotation?: (token: string) => unknown;
```

Comment:
This callback doesn't really do any "handling" of the rotation, right? And there's nothing we're doing with the return value?

```suggestion
  onNewRefreshToken?: (newToken: string) => void;
```

Reply:
I see you left the return type to be `unknown`, implying that SCAN is going to do something with that return type. Is there a reason for not making it `void`, or did you just overlook that? :)